### PR TITLE
Remove random references to siblings.

### PR DESCRIPTION
**Pull request by iHiD**
See https://forum.exercism.org/t/simple-cipher-exercise-description/2597 for discussion.

There is zero context for this having anything to do with siblings, so let's stick to the initial theme about Caesar.

### CODE REVIEW

**Review by SleeplessByte — APPROVED** [review]

**Review by kotp — APPROVED** [review]
Approving with a proposed change.

**Review by kytrinyx — APPROVED** [review]
❤️

**Review by IsaacG — APPROVED** [review]

File: exercises/simple-cipher/description.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ When "ldpdsdqgdehdu" is put into the decode function it would return the origina
 
 ## Step 2
 
-Shift ciphers are no fun though when your kid sister figures it out.
+Shift ciphers are no fun though when the opposition commander figures it out.
 Try amending the code to allow us to specify a key and use that for the shift distance.
 This is called a substitution cipher.
```
Comment (iHiD) [inline exercises/simple-cipher/description.md:34]:
```suggestion
Shift ciphers quickly cease to be useful when the opposition commander figures them out.
So instead, let's try using a substitution cipher.
Try amending the code to allow us to specify a key and use that for the shift distance.
```

**Comment by iHiD** [comment]
Thanks all :)

**Pull request merged**